vm_core.h

```diff
@@ -221,11 +221,18 @@ struct iseq_inline_cache_entry {
     rb_serial_t ic_serial;
     const rb_cref_t *ic_cref;
     union {
```

Comment:
why union?

Reply:
Only to keep diff smaller.  There is no need for the union (of course).  I can remove it.